src/MBrace.Flow/CloudFlow.fs

```diff
@@ -1123,3 +1139,55 @@ module CloudFlow =
                         return! projection collector.Result
                 } }
 
+
+
+
+                
+    /// <summary>Creates a CloudFlow from the ReceivePort of a CloudChannel</summary>
+    /// <param name="channel">the ReceivePort of a CloudChannel.</param>
+    /// <returns>The result CloudFlow.</returns>
+    let ofCloudChannel (channel : IReceivePort<'T>) : CloudFlow<'T> =
```

Comment:
Yes I agree a more explicit CloudFlow.ofCloudSource
```
CloudFlow.ofCloudSource(source : ICloudSource<'T>, degreeOfPar : int) : CloudFlow
``` 
is certainly better than the implicit.